components/templating.rst

```diff
@@ -139,9 +139,8 @@ Helpers
 
 The Templating component can be easily extended via helpers. Helpers are PHP objects that
 provide features useful in a template context. The component has
-2 built-in helpers:
+a built-in helper:
```

Comment:
"The component has one helper built-in:" ? (I'm not a native, but it sounds a bit better to me)